README.md

```diff
@@ -42,9 +42,9 @@ Operators can behave like managed service providers. Their user interface on the
 
```

Comment:
changed moved to : https://github.com/operator-framework/operator-lifecycle-manager/pull/1480